dev-tools/scripts/smokeTestRelease.py

```diff
@@ -1223,7 +1223,7 @@ def run_java(cmd, logfile):
   return jc(run_java8, java8_home, run_java9, java9_home)
 
 version_re = re.compile(r'(\d+\.\d+\.\d+(-ALPHA|-BETA)?)')
```

Comment:
If you're using the short hash version then the previous line was correct (+ instead of fixed-character count 11)?

Reply:
This regex extracts the hash from the filename, which still has a full sha. It looks something like `$TMPDIR/releases-8.10/lucene-solr-8.10.0-RC3-revb473a4f88588bb1e32cff27aaee05380c42f64f1`

Reply:
Ah, ok, clear, thanks!